pkgs/by-name/mi/miru/update.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl jq gnused
+
+set -euo pipefail
+
+ROOT="$(dirname "$(readlink -f "$0")")"
+if [[ ! "$(basename $ROOT)" == "miru" || ! -f "$ROOT/package.nix" ]]; then
+    echo "ERROR: Not in the miru folder"
```

Comment:
Oops sorry I missed this one, same as the previous error message
```suggestion
    echo "ERROR: Not in the miru folder" >&2 
```

Reply:
LGTM after this one !

Reply:
@matteo-pacini You re-requested a review but did not fix this last issue, was this a mistake?

Reply:
@d4ilyrun yup, my bad! Forgot to push the changes 😇